packages/@lwc/engine-core/src/framework/hydration.ts

```diff
@@ -165,9 +166,11 @@ function getValidationPredicate(
     optOutStaticProp: string[] | true | undefined
 ): AttrValidationPredicate {
     // `data-lwc-host-mutated` is a special attribute added by the SSR engine itself,
-    // which does the same thing as an explicit `static validationOptOut = true`.
-    if (renderer.getAttribute(elm, 'data-lwc-host-mutated') === '') {
-        return (_attrName: string) => false;
+    // which does the same thing as an explicit `static validationOptOut = ['attr1', 'attr2']`.
+    const hostMutatedValue = renderer.getAttribute(elm, 'data-lwc-host-mutated');
+    if (isString(hostMutatedValue)) {
+        const mutatedAttrValues = new Set(StringSplit.call(hostMutatedValue, / /));
```

Comment:
`/ /` looks weird...

Reply:
Sorry, too late. I did not see your comments before merging.

TypeScript for some weird reason complains if I use `' '`. So I used `/ /`.

Reply:
Well I was too slow to review. I bet it's because of our weird re-mapping losing overloaded function signatures.